share/tools/web_config/webconfig.py

```diff
@@ -1,48 +1,31 @@
-from __future__ import unicode_literals
-from __future__ import print_function
 import binascii
-
-try:
-    from html import escape as escape_html
-except ImportError:
-    from cgi import escape as escape_html
 import errno
 import glob
+import http.server as SimpleHTTPServer
+import json
 import multiprocessing.pool
 import operator
 import os
 import platform
 import re
 import select
 import socket
+import socketserver as SocketServer
 import subprocess
 import sys
 import tempfile
 import threading
-from typing import Optional
+from html import escape as escape_html
 from itertools import chain
+from typing import Optional
+from urllib.parse import parse_qs
 
 COMMON_WSL_CMD_PATHS = (
     "/mnt/c/Windows/System32",
     "/windir/c/Windows/System32",
     "/c/Windows/System32",
 )
 FISH_BIN_PATH = False  # will be set later
-IS_PY2 = sys.version_info[0] == 2
-
-if IS_PY2:
```

Comment:
Right, this means we can remove a everything <3.5 from `share/functions/__fish_anypython.fish`